content/blog/2019-09-23-grpc-on-dotnetcore.md

```diff
@@ -0,0 +1,61 @@
+---
+author: Sourabh Shirhatti
+author-link: https://twitter.com/sshirhatti
+date: "2019-09-23T00:00:00Z"
+published: true
+title: .NET Core ❤ gRPC
+url: blog/grpc-on-dotnetcore
+---
+
+_This is a guest post by [Sourabh Shirhatti](https://twitter.com/sshirhatti), a Program Manger on the .NET team at Microsoft._
+
+The .NET team at Microsoft has been working in close collaboration with the gRPC team since November 2018 on a new fully managed implementation of gRPC for .NET Core.
+
+We're pleased to announce that **grpc-dotnet** is now available with .NET Core 3.0 today!
+
+## How to get it?
+
+**grpc-dotnet** packages have just been released to [NuGet.org](https://www.nuget.org/profiles/grpc-packages) and are already available for use in your projects. These packages also require the latest .NET Core 3.0 shared framework. You can download the .NET Core 3.0 SDK for your dev machine and build servers from the [.NET Core 3.0 download page](https://aka.ms/netcore3download) to acquire the shared framework. 
+
+## Getting Started
+
+gRPC templates are included as part of the .NET SDK. To get started, navigate to a console window after installing the SDK and run the following commands.
```

Comment:
optional suggestion:  As a .NET user, now seeing gRPC project template being shipped as integral part of .NET SDK is a big deal and it's a milestone to be excited about. IMHO the sentence "gRPC templates are included as part of the .NET SDK." comes out a as bit dry formulation of what many will consider big news. I wouldn't be afraid of trying to make that sentence sound a bit more exciting.  (sth in the sense - "That's right, this is the first time gRPC becomes a first-class citizen in the .NET ecosystem and is a technology endorsed by Microsoft" - you get the point). This is just an idea, leaving up to you.

Reply:
I've made sure to use the phrase first-class citizen 😄